packages/compile-common/src/shims.ts

```diff
@@ -1,4 +1,4 @@
-import { LinkReference, CompiledContract } from "./index";
+import { Bytecode, LinkReference, CompiledContract } from "./index";
```

Comment:
We should get these types into a separate `types.ts` file, otherwise we'll run the risk of circular imports. (This can probably be on the other PR, though)